maps/_templates/area_estimate_intercomparison.ipynb

```diff
@@ -0,0 +1,644 @@
+{
```

Comment:
<p>Why is this necessary? Is it intended to be run from Google Colab?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/crop-mask/pull/391/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Yes, Google Colab for now.
I'll add modifications for local later.